Backend-API/src/main/java/com/avangers/backendapi/controllers/UserController.java

```diff
@@ -33,4 +30,16 @@ public ResponseEntity<?> register(@Valid @RequestBody RegisterUserRequestDTO reg
       return new ResponseEntity<>(error, HttpStatus.BAD_REQUEST);
     }
   }
+
+  @PutMapping("/api/users/{userId}")
```

Comment:
we'll only update the logged-in user, right? @markkovari is it possible to use Principal inside parameters?

Reply:
Yesyes, this is the way 
![image](https://github.com/user-attachments/assets/4eee4622-3c25-4bb0-82ab-1dee9e0db0a4)